### PR TITLE
Feature/#6140 add toolbar menu dismiss setter

### PR DESCRIPTION
**Pull request by BranescuMihai**
---

### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### After merge
- [ ] **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- [ ] **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -22,6 +22,9 @@ permalink: /changelog/
 
 * **feature-accounts-push**
   * Add known prefix to FxA push scope.
+  
+* **browser-toolbar**
+  * Add possibility to listen to menu dismissal through `setMenuDismissAction` in `DisplayToolbar`
```
Comment (Amejia481) [inline docs/changelog.md]: nit: Add **the** possibility ....

Comment (BranescuMihai) [reply docs/changelog.md]: done!

**Comment by BranescuMihai** [comment]
bors r=Amejia481

**Comment by Amejia481** [comment]
bors retry

**Comment by BranescuMihai** [comment]
bors retry

**Pull request merged**